odyssey-tmp/org.coolstar.safemode_1.1.2-2_iphoneos-arm.deb

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-type" content="text/html; charset=utf-8">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; style-src 'unsafe-inline'; img-src data:; connect-src 'self'">
+    <title>Page not found &middot; GitHub Pages</title>
```

Comment:
I'm not going to add a deb file to the repo, as it's meant to fetch the latest ones, but... this is HTML.